angular/package.json

```diff
@@ -49,41 +49,38 @@
     "tslib": "^1.9.3"
   },
   "peerDependencies": {
-    "@angular-devkit/core": "^7.2.1",
-    "@angular-devkit/schematics": "^7.2.1",
-    "@angular/core": "^7.2.1",
-    "@angular/common": "^7.2.1",
-    "@angular/forms": "^7.2.1",
-    "@angular/router": "^7.2.1",
-    "@angular/compiler": "^7.2.1",
-    "@angular/compiler-cli": "^7.2.1",
-    "@angular/platform-browser": "^7.2.1",
-    "@angular/platform-browser-dynamic": "^7.2.1",
-    "rxjs": ">=6.2.0",
-    "zone.js": "^0.8.26"
+    "@angular/common": "~8.0.0",
+    "@angular/compiler": "~8.0.0",
+    "@angular/core": "~8.0.0",
+    "@angular/forms": "~8.0.0",
+    "@angular/platform-browser": "~8.0.0",
+    "@angular/platform-browser-dynamic": "~8.0.0",
+    "@angular/router": "~8.0.0",
+    "rxjs": "~6.4.0",
```

Comment:
I would suggest not to be so strict. I think    `"rxjs": ">=6.4.0" `would be better (ionic-native expects 6.5.0 or newer).

Same for Angular. I think `"@angular/common": ">=7.2.1"`, would be better if Angular 7 is still supported, otherwise `"@angular/common": ">=8.0.0"`

Reply:
Good Point. Angular 8 is backwards compatible to angular 7 and should be allowed.

Reply:
For rxjs, `~6.4.0` will still allow for 6.5. Please see the semver check below.

https://jubianchi.github.io/semver-check/#/~6.4.0/6.5.0

For angular, it is not backwards compatible. There are breaking changes between 7 and 8, which is why the versions are set the way they are.

Reply:
@mhartington no, 6.5.0 does not satisfy constraint ~6.4.0 - you have to choose ^6.4.0 - then it will satisfy 6.5.0